### PR TITLE
[release-4.11] Fix namespace store tests

### PR DESCRIPTION
**Pull request by vbnrh**
Signed-off-by: Vineet Badrinath <vineetbnath@gmail.com>

### CODE REVIEW

**Comment by vbnrh** [comment]
Screenshot from 2022-10-07 15-40-06
Screenshot from 2022-10-07 15-30-16
Screenshot from 2022-10-07 15-26-05
Screenshot from 2022-10-07 14-45-31
Screenshot from 2022-10-06 16-42-49
Screenshot from 2022-10-06 16-18-42
Screenshot from 2022-10-06 15-47-27
Screenshot from 2022-10-06 15-41-51
Screenshot from 2022-10-06 15-37-50

File: cypress/views/block-pool.ts [hunk 1]
```diff
@@ -75,8 +75,11 @@ export const verifyFooterActions = (action: string) => {
 export const verifyBlockPoolJSON = (
   compressionEnabled: boolean = true,
   replica: string = replicaCount
-) =>
-  cy.exec(`oc get cephBlockPool ${poolName} -n  ${NS} -o json`).then((res) => {
+) => {
+  cy.wait(5000);
```
Comment (bipuladh) [inline cypress/views/block-pool.ts:79]: Why arbitrary wait?

Comment (bipuladh) [reply cypress/views/block-pool.ts:79]: I don't this being required in master and the tests are passing fine.

**Review by bipuladh — APPROVED** [review]

File: cypress/views/block-pool.ts (hunk 1 above)
Comment (vbnrh) [reply cypress/views/block-pool.ts:79]: unsure but the times i've ran it. The resource did not get detected, maybe my cluster servers are slow and CI ones are fast enough to quickly register it.

**Pull request merged**